tasks/utils/news.ts

```diff
@@ -0,0 +1,14 @@
+// import { getConfig } from './config'
+import type { LocalizedNews } from '@helpwave/common/util/news'
+import { localizedNewsSchema } from '@helpwave/common/util/news'
+
+export const fetchLocalizedNews = (): Promise<LocalizedNews> => {
+  // TODO change later back
+  const { featuresFeedUrl } = { featuresFeedUrl: 'https://cdn.helpwave.io/feed_intl.json' }// getConfig()
+  return fetch(featuresFeedUrl)
+    .then((res) => res.json())
+    .then(async (json) => {
+      await localizedNewsSchema.parseAsync(json)
```

Comment:
why are you using `.parseAsync` here?

this could just be the following:

```ts
.then((json) => {
  const maybeLocalizedNews = localizedNewsSchema.safeParse(json)
  if (!maybeLocalizedNews.success) {
    // TODO: throw custom error with a useful message
  } else {
    return maybeLocalizedNews.data
  }
})
```

You should return `maybeLocalizedNews.data` here as this has the transforms applied

Reply:
Due to the fact that `getServerSideProps` wants to return a serializable struct. This was not possible with the date inside. I don't know if `.date` solves that, this would be great, and we don't have to reparse this again.

Reply:
Oh I see, didn't consider this. In that case this makes sense

`.date` wouldn't solve this

Reply:
Yes. I've tried this out right now .. :)
```
Error: Error serializing `.localizedNews.de[0].date` returned from `getServerSideProps` in "/".
Reason: `object` ("[object Date]") cannot be serialized as JSON. Please only return JSON serializable data types.
```